src/test/java/core/basesyntax/service/FileServiceTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class FileServiceTest {
+    private static final String INPUT_FILE_NAME = "src/test/resources/input.csv";
+    private static FileService fileService;
+
+    @BeforeAll
+    static void beforeAll() {
+        fileService = new FileService();
+    }
+
+    @Test
+    void readFromFile_nullFileName_notOk() {
+        Assertions.assertThrows(NullPointerException.class, () -> fileService.readFromFile(null));
+    }
+
+    @Test
+    void writeToFile_nullFileName_notOk() {
+        Assertions.assertThrows(NullPointerException.class,
+                () -> fileService.writeToFile(null, ""));
+    }
+
+    @Test
+    void readFromFile_notExistingFileName_notOk() {
+        Assertions.assertThrows(RuntimeException.class, () -> fileService.readFromFile(""));
+    }
+
+    @Test
+    void writeToFile_notExistingFileName_notOk() {
+        Assertions.assertThrows(RuntimeException.class, () -> fileService.writeToFile("", ""));
+    }
+
+    @Test
+    void readFromFile_existingFileName_Ok() {
```

Comment:
```suggestion
    void readFromFile_existingFileName_ok() {
```
use lowercase after underscore